app/controllers/rentals_controller.rb

```diff
@@ -0,0 +1,22 @@
+class RentalsController < ApplicationController
+
+  def create
+    rental = Rental.new(rental_params)
+    rental.movie = Movie.find_by title: params[:title]
+    rental.check_out = Date.today
+     if rental.movie.available_inventory == 0
+       render status: :bad_request, json: { error: "Not enough inventory" }
+     elsif rental.save
+       render status: :ok, json: { id: rental.id }
+       rental.customer.movies_checked_out_count += 1
```

Comment:
If the rental is processed, the customer.movies_checked_out_count increments up one, which makes sense.  Does the movie.available_inventory also need to decrease by one?